app/furniture/marketplace/tags_controller.rb

```diff
@@ -42,6 +42,20 @@ def index
       skip_authorization
     end
 
+    def destroy
+      authorize(mtag).destroy
+
+      respond_to do |format|
+        format.turbo_stream do
+          if mtag.destroyed?
+            render turbo_stream: turbo_stream.remove(mtag)
+          else
+            render turbo_stream: turbo_stream.replace(mtag)
+          end
+        end
```

Comment:
Do we need a turbo stream? I thought we had gotten to the point where we can just render plain html and Turbo will do a diff-and-swap within the marketplace?

Reply:
Ah right. I'll double check.